inc/Point2D.h

```diff
@@ -2,13 +2,8 @@
 
 class Point2D
 {
-	// TODO 1:
-	// 1. Define a constructor that is public, otherwise you wont be
-	//	  able to access it outside the class
-	// 2. Define 2 public member variables to store x & y co-ordiinates
-	//	  It is a convention that member variables have a trailing
-	//    underscore. For example, someVariable_ instead of someVariable
-	//	  Or some people will write it as m_someVariable but not preffered
-
-	// The function implimentation must be in the cpp file
-};
\ No newline at end of file
+public:
+	int x_, y_;
+	Point2D();
+	Point2D(int x, int y);
```

Comment:
Even though the code will work fine without the argument names, it is better to include them. Say for example, you have a function that takes 3 arguments of type double: initial amount, interest rate and duration. If the function is declared something like this: `compute_interest(double, double, double)`, what is gonna happen is that when ever you include it in other packages(files), the IDE is gonna suggest the same `compute_interest(double, double, double)`. It is impossible to tell in what order should the arguments be passed. On the contrary, this is quite preferable: `compute_interest(double initial_amount, double interest_rate, double duration)`